app/src/ingest_policy_pdfs.py

```diff
@@ -14,17 +20,114 @@
 
 
 def _ingest_policy_pdfs(
-    db_session: db.Session,
     pdf_file_dir: str,
     doc_attribs: dict[str, str],
 ) -> None:
     file_list = get_files(pdf_file_dir)
     embedding_model = app_config.sentence_transformer
+    db_session = app_config.db_session()
     for file in file_list:
         if file.endswith(".pdf"):
-            logger.info(
-                f"Processing pdf file: {file} at {pdf_file_dir} using {embedding_model}, {db_session}, with {doc_attribs}"
+            with smart_open_file(file, "rb") as fin:
+                output_string = extract_text(fin, laparams=LAParams())
+                logger.info(
+                    f"Processing pdf file: {file} at {pdf_file_dir} using {embedding_model}, {db_session}, with {doc_attribs}"
```

Comment:
I would put this logger line before `extract_text()` in case there are errors when calling the function or maybe even before `smart_open_file` in case there are problems accessing the file, for example from S3.
What's the usefulness of logging `db_session`? 
I'd log `pdf_file_dir`, `embedding_model`, and `doc_attribs` outside of the `for file` loop since it doesn't change for each file.